### DISCUSSION TITLE
What can cause a sudden and significant speed decrease?

### DISCUSSION THREAD

**Discussion by jswag245** [lllyasviel/stable-diffusion-webui-forge#2473] [General]
I've noticed that very rarely, my speed will drop from 6 s/it for one generation to about 18 s/it for the following generation. It will remain that this lower speed until I restart my laptop. Any ideas why this issue is occurring?

**Comment by logoth80** [comment]
there can be too many reasons to count, but if I were to bet, you filled RAM or VRAM. 
make sure you use 'OOM prevention' and watch for excessive tabs in browser. If restarting FORGE doesn't help and you need system restart, then it's probably some other app taking resources.

**Reply by jswag245** [reply]
I would've assumed that I would've at least received an error message or an OOM message. I don't open any additional tabs or apps during generation. I'll keep monitoring in the meantime, very strange.

**Reply by jswag245** [reply]
Update: My RAM and VRAM usage was not maxed out. It became close, like 14gb out of 16gb and 4.1gb out of 6gb but nothing crazy. I normally experience my faster speeds at that usage.
